massa-models/src/config/constants.rs

```diff
@@ -225,6 +225,10 @@ pub const NETWORK_NODE_EVENT_CHANNEL_SIZE: usize = 10_000;
 //
 /// Threshold to accept a new versioning
 pub const VERSIONING_THRESHOLD_TRANSITION_ACCEPTED: Amount = Amount::from_mantissa_scale(75, 0);
+/// versioning controller event channel size
+pub const VERSIONING_CONTROLLER_CHANNEL_SIZE: usize = 10_000;
+/// versioning block stats queue size
+pub const VERSIONING_NB_BLOCKS_CONSIDERED: usize = 100;
```

Comment:
Looking for the previous 100 blocks seems quite low no?

Reply:
Indeed, I think something closer to 1000 (or even 10000) would be more appropriate, which would only be 8 minutes worth of blocks for 1000. Maybe we could model the statistics involved.